applications/luci-app-dcwapd/files/usr/lib/lua/luci/model/cbi/dcwapd/dcwapd.lua

```diff
@@ -0,0 +1,237 @@
+dat_ssid.rmempty = false
+dat_ssid.size = 0
+for i, v in ipairs(devices) do
+    ntm.init()
+    local net = ntm:get_wifinet(v)
+    if net then
+        dat_ssid:value(net:ssid())
+        dat_ssid.size = dat_ssid.size + 1
+    end
+end
+
+-- Data bridge option
+dat_bridge = s:option(Value, "bridge", translate("Bridge"))
+dat_bridge.optional = false
+dat_bridge.rmempty = false
+dat_bridge.size = 0
+for i, v in ipairs(devices) do
+    ntm.init()
+    local net = ntm:get_wifinet(v)
+    if net then
+        local nw = net:get_network()
+        if nw then
+            dat_bridge:value("br-" .. nw:name())
+            dat_bridge.size = dat_bridge.size + 1
+        end
+    end
+end
+
+-- Data interfaces list
+ifaces = s:option(MultiValue, "interfaces", translate("Interfaces"))
+ifaces.widget = "checkbox"
+ifaces.optional = true
+ifaces.rmempty = false
+--ifaces.size = #devices + 1
+ifaces.size = 0
+--ifaces:value("")
+table.sort(devices)
+for i, v in ipairs(devices) do
+    ntm.init()
+    local net = ntm:get_wifinet(v)
+    if net then
+        ifaces:value(v)
+        ifaces.size = ifaces.size + 1
+    end
+end
+
+-- Filter sets section
+s = m:section(TypedSection, "filter-set", "Filter Sets", "Filter Sets")
+s.addremove = true
+s.dynamic = false
+s.anonymous = false
+s.optional = false
+
+-- MAC address option
+mac = s:option(Value, "mac", translate("MAC Address"))
+mac.optional = false
+mac.rmempty = false
+
+-- Filters list
+filters = s:option(MultiValue, "filters", translate("Filters"))
+filters.widget = "checkbox"
+filters.optional = false
+filters.rmempty = false
+filters.size = 0
+uc:foreach("dcwapd", "filter",
+    function(s)
+        if s['.name'] then
+            filters:value(s['.name'])
+            filters.size = filters.size + 1
+        end
+    end)
+
+-- Filters section
+s = m:section(TypedSection, "filter", "Filters", "Filter Rules")
+s.template = "cbi/tblsection"
+s.anonymous = false
+s.addremove = true
+s.sortable = true
+
+-- Packet Size
+packetsize = s:option(Value, "packet_size", translate("Packet size"))
+packetsize.rmempty = false
+packetsize:value("*", translate("*"))
+packetsize.default = translate("*")
+
+-- Source IP
+srcip = s:option(Value, "source_ip", translate("Source IP"))
+srcip.rmempty = false
+srcip:value("*", translate("*"))
+srcip.default = translate("*")
+--wa:cbi_add_knownips(srcip)
+
+-- Source Port
+srcport = s:option(Value, "source_port", translate("Source port"))
+srcport.rmempty = false
+srcport:value("*", translate("*"))
+srcport.default = translate("*")
+
+-- Protocol
+proto = s:option(Value, "protocol", translate("Protocol"))
+proto:value("*", translate("*"))
+proto:value("tcp", "TCP")
+proto:value("udp", "UDP")
+proto:value("icmp", "ICMP")
+proto.rmempty = false
+proto.default = translate("*")
+
+-- Destination Port
+dstport = s:option(Value, "dest_port", translate("Destination port"))
+dstport.rmempty = false
+dstport:value("*", translate("*"))
+dstport.default = translate("*")
+
+return m
```

Comment:
Please remove the lines on config option that are comments
And also please make the variables for the uci values local on the top of this file
`local dstport, proto,`  and so on